common/plugins/com.liferay.ide.ui/plugin.xml

```diff
@@ -182,8 +182,8 @@
             id="com.liferay.ide.ui.workspace.launchWorkspace"
             name="Launch Workspace">
             <commandParameter
-                  id="com.liferay.ide.ui.workspace.launchWorkspace.workspaceLocation"
-                  name="Workspace Location"
+                  id="workspaceLocation"
```

Comment:
I read the source code dealing with the menu issues and make sure it it necessary  to do this